src/data-table/index.js

```diff
@@ -342,7 +342,7 @@ class DataTable extends Component {
               {
                 columns.length === 0 &&
                 <Table.Row textAlign='center'>
-                  <Table.HeaderCell colSpan={columns.length}>{noColumnsMsg}</Table.HeaderCell>
+                  <Table.HeaderCell colSpan={columns.length}>{data.length === 0 ? 'Empty data :(' : noColumnsMsg}</Table.HeaderCell>
```

Comment:
Should we expose this message as one of the configurable props like `emptySearchMsg` and `noColumnsMsg` (but with this current one as default)?